### PR TITLE
basic html markup

### PR DESCRIPTION
**Pull request by kimitaWanjohi**
- Build the basic structure for the application

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,14 +1,67 @@
...
+              placeholder="Your name"
+            />
+            <input
+              class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-blue-500 focus:shadow-outline"
+              id="score"
+              type="text"
+              placeholder="Your score"
+            />
+            <button class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded">
+              Submit
```
Comment (Shedrack-Sunday) [inline public/index.html:58]: - [x] Consider adding a `start` in your `script`, so a start command can start the project automatially. 
- For example:

```
 "scripts": {
    "start": "webpack --config webpack.config.js --watch"
  }
```

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @kimitaWanjohi     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
- Webpack setup properly✔️
- Linters passing ✔️
- Correct GitHub flow  
  
# Required Changes ♻️

- I did run your code successfully, but add a start build to run it automatically 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: public/index.html (hunk 1 above)
Comment (kimitaWanjohi) [reply public/index.html:58]: sure I'll although I already had webpack-dev-server doing that.

**Review by binhussen — APPROVED** [review]
Hi @kimitaWanjohi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**